content/blog/zen.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "The Zen of CoreDNS"
+description = "Philosophy behind CoreDNS"
+tags = ["Zen", "Meta"]
+draft = false
+date = "2019-09-28T13:55:25Z"
+author = "CoreDNS Authors"
++++
+
+CoreDNS is a DNS server, written in Go. It links *plugins* to provide a customized DNS service.

```

Comment:
May be worth adding a link to https://coredns.io/plugins/  -- `It links [plugins](https://coredns.io/plugins/) to provide...`